src/frontend/src/common/images/logo.svg

```diff
@@ -1,12 +1 @@
-  <path d="m250.9 55.53h-13.521v18.281h-5.864v-49.008h19.385c9.301 0 16.094 6.8554 16.094 15.363 0 8.5077-6.793 15.363-16.094 15.363zm-13.521-25.594v20.461h13.521c5.72 0 10.23-4.5098 10.23-10.23s-4.51-10.23-10.23-10.23h-13.521z" clip-rule="evenodd" fill="#fff" fill-rule="evenodd"/>
-  <path d="m284.73 50.777v17.901h28.884v5.1328h-34.748v-49.008h34.017v5.1328h-28.153v15.708h22.303v5.1328h-22.303z" fill="#fff"/>
-  <path d="m332.64 34.779v39.032h-5.864v-49.008h5.322l30.143 39.032v-39.032h5.864v49.008h-5.321l-30.144-39.032z" fill="#fff"/>
-  <path d="m93.5 71.69c-6.3518 0-11.5 5.064-11.5 11.311s5.1477 11.31 11.5 11.31c6.3528 0 11.5-5.063 11.5-11.31s-5.1477-11.311-11.5-11.311z" fill="#93C151"/>
-  <path d="m59.056 40.715 14.037-29.628 25.273 54.198c-1.6225-0.464-3.3384-0.7128-5.1135-0.7128-8.0438 0-14.872 5.1088-17.303 12.204-0.3691-0.6778-0.7191-1.3756-1.0521-2.0896l-15.841-33.972z" fill="#fff"/>
-  <path d="m66.716 3.8319c-9.2833-4.6548-16.398-4.9548-21.813-1.5188-3.3213 2.1059-5.7211 5.4543-7.5473 9.3708l-36.988 79.321 1.742 0.8735c8.8201 4.4229 15.688 4.9209 21.019 1.9898 1.4356-0.7889 2.711-1.803 3.8474-2.9923 1.8126-1.8975 3.2775-4.24 4.4943-6.8495l36.988-79.321-1.7421-0.87349z" fill="#fff"/>
-</svg>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 362.52 67.41"><defs><style>.cls-1{fill:#ff9c06;}.cls-2,.cls-3{fill:#fff;}.cls-3{font-size:50.52px;font-family:TTNorms-ExtraBold, TT Norms;font-weight:800;}.cls-4{font-family:TTNorms-Regular, TT Norms;font-weight:400;}</style></defs><g id="SIZE_MARKER" data-name="SIZE MARKER"><circle class="cls-1" cx="57.65" cy="8.1" r="8.1"/><path class="cls-2" d="M5.31,22.14a21.63,21.63,0,0,1,7.93-1.34H21.4V16.2H47.3a12.86,12.86,0,0,1,0-16H18.55a14.62,14.62,0,0,0-8,1.93C7.24,4.24,5.27,8.31,5.31,15.37l0,.83h0Z"/><path class="cls-2" d="M60.41,51.4H16.09V41.8h31A17.59,17.59,0,0,0,52.72,41a10.44,10.44,0,0,0,2.86-1.46c3.13-2.28,4.93-6.52,4.73-13.73H13.24a14.71,14.71,0,0,0-8,1.93C1.81,29.92-.2,34.22,0,41.79H0V67.41H47.18a17.88,17.88,0,0,0,5.64-.81,10.75,10.75,0,0,0,2.86-1.47C58.81,62.85,60.61,58.61,60.41,51.4Z"/><text class="cls-3" transform="translate(79.69 51.83)">CZ <tspan class="cls-4" x="81.83" y="0">GEN EPI</tspan></text></g></svg>
```

Comment:
since we have multiple logos should we rename this filename to be more descriptive (like for example we have the long logo (CZ Gen Epi) and then we also have the favicon looking E and then we also have the white and black variations), just think it might be nice to have the filename tell us more about what type of logo this is.

Reply:
So for this particular one maybe we could rename to `FullLogoWhite.svg`?

Reply:
Good point! Changed it, thanks